src/Components/Button.jsx

```diff
@@ -1,7 +1,32 @@
-const Calculator = () => (
-  <div>
-    calculator
+import './button.css';
+
+const Button = () => (
```

Comment:
- [ ] This is the only component you have, but you should have a Calculator component with at least one child component. Here you have a Button component with no child component. Kindly refer to the requirements and fix this issue. Kindly create a component called Calculator and ensure it has at least one child component.